### PR TITLE
Updates docs

### LINKED ISSUE #7: AboutUs - Heinrich
Update AboutUs by Heinrich

### LINKED ISSUE #12: Update User Guide - Quick Start
Update User Guide for Quick Start.

### LINKED ISSUE #21: Developer Guide - Target user profile, value proposition, and user stories

### LINKED ISSUE #23: Developer Guide - Non-functional requirements

### PR DESCRIPTION
**Pull request by HynRidge**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ title: About Us
 
 ### Lu Luoyi
 
+
```
Comment (luo-git) [inline docs/AboutUs.md]: Should there be an extra white line here?

File: docs/AboutUs.md [hunk 2]
```diff
@@ -27,3 +28,11 @@ title: About Us
...
 
 * Role: Developer
+
+### Heinrich
+<img src="images/HynRidge.png" width="200px">
+
+[github](https://github.com/HynRidge)
+
+
+<
```
Comment (luo-git) [inline docs/AboutUs.md]: Maybe there shouldn't be a `<` here?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -236,36 +236,43 @@ _{Explain here how the data archiving feature will be implemented}_
...
-*{More to be added}*
 
 ### Use cases
 
 (For all use cases below, the **System** is the `HelloFile` and the **Actor** is the `user`, unless specified otherwise)
 
 <br />
 
+
+<br />
```
Comment (anthony6401) [inline docs/DeveloperGuide.md]: Why is there another `<br />`? Is it intended?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -464,39 +479,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-		
+				
 * *a. HelloFile has no permission to read or write to the file system.
 
 	* *a1. HelloFile prompts the user that it needs the read and write permission.
 	
-	    Use case resumes.
-
+		    Use case resumes.
+
```
Comment (luo-git) [inline docs/DeveloperGuide.md:508]: I think you broke some of my parts. You can review this markdown file again to find out more.

Before
image

After
image

File: docs/AboutUs.md [hunk 5]
```diff
@@ -27,3 +28,11 @@ title: About Us
 [github](https://github.com/Anthony6401)
 
 * Role: Developer
+
+### Heinrich
+<img src="images/HynRidge.png" width="200px">
+
+[github](https://github.com/HynRidge)
+
+
```
Comment (anthony6401) [inline docs/AboutUs.md]: I think you forget to add role

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (luo-git) [reply docs/DeveloperGuide.md:508]: I have night mode enabled so it is more obvious.

**Review by luo-git — CHANGES_REQUESTED** [review]
Good job on the formatting of the user stories.
I have noticed that you have changed some of the spacings. Is there a reason for that?
Apart from that, just need some minor changes and this will be good to merge. =)

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -509,23 +524,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-	
-	* *a1. HelloFile shows the error message and direct the user to enter the user help command.
-	
-	    Use case ends.
 
+	* 2a1. HelloFile shows the error message.
+		
+	    Use case resumes from step 1.
+	        
+* *a. HelloFile has no permission to read or write to the file system.
```
Comment (anthony6401) [inline docs/DeveloperGuide.md]: I think HelloFile does not need to read or write in help command, it shouldn't be in extension.

**Review by anthony6401 — CHANGES_REQUESTED** [review]
A few nits to fix.

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (anthony6401) [reply docs/DeveloperGuide.md]: This is the same as Luo yi's comment at the top.

Comment (HynRidge) [reply docs/DeveloperGuide.md]: should I remove the whole extension?

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (HynRidge) [reply docs/DeveloperGuide.md:508]: Okay, I have restore to original, sorry for the different input. I merge code to master and there were conflicts.

File: docs/AboutUs.md (hunk 2 above)
Comment (HynRidge) [reply docs/AboutUs.md]: Yep.. resolved

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -362,35 +370,40 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 	* *a1. HelloFile prompts the user that it needs the read and write permission.
-	
+		
 	    Use case resumes.
+	    
+	    
+<br />
+
 
 <br />
```
Comment (Ma-Yueran) [inline docs/DeveloperGuide.md]: Should there be 2 br ?

**Review by Ma-Yueran — COMMENTED** [review]
Good work! it's quite difficult to make the format perfect. I found a small problem of using an extra br, but other things looks good to me

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (HynRidge) [reply docs/DeveloperGuide.md]: not sure

Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: I think between other points it's 1 br, but not a big problem

**Review by Ma-Yueran — APPROVED** [review]
Looks good to me

**Review by anthony6401 — APPROVED** [review]
Looks good to merge!

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (luo-git) [reply docs/DeveloperGuide.md:508]: There's still something wrong here.

image

**Review by luo-git — CHANGES_REQUESTED** [review]
Still broken. Require some changes.

**Comment by HynRidge** [comment]
@luo-git I have made the changes. Could you kindly check again? Sorry for the overhead confussion!

**Review by luo-git — APPROVED** [review]
LGTM

**Pull request merged**